Cargo.toml

```diff
@@ -46,6 +46,7 @@ cargo_metadata = "0.18"
 clap = { version = "4", features = ["derive", "env", "string"] }
 clap-verbosity-flag = "2"
 console = "0.15"
+darling = "0.20.3"
```

Comment:
Not sure if we really need this, but I would need help from someone more fluent in rust macros to implement it without this helper.

Reply:
Let's go with it. Otherwise, you would have to parse tokens manually, which would be a nightmare.